website/docs/about/accessibility-statement.md

```diff
@@ -1,49 +1,47 @@
 ---
-title: Accessibility Statement
-description: This is an accessibility statement from the Design Systems Team.
-caption: Learn more about the accessibility intentions of the design system.
+title: Accessibility statement
 order: 104
 ---
 
-Accessibility is a core requirement, not an optional feature; this aligns with our [company principles](https://www.hashicorp.com/our-principles), especially our principle of integrity.
+Accessibility is a core requirement and not an optional feature. Our approach to accessibility closely aligns with our [HashiCorp principles](https://www.hashicorp.com/our-principles), such as our core principle of integrity.
 
 > Integrity requires a consistency of our thoughts, words, and actions and a dedication to the truth.
 
-To this end, we have built the Helios Design System with accessibility as a core requirement. In this way, we are giving HashiCorp product designers and developers a greater ability to deliver accessible product experiences for all HashiCorp users.
+To this end, we have built the Helios Design System with accessibility as a core requirement. In this way, we’re giving HashiCorp product designers and engineers a greater ability to deliver accessible product experiences for all HashiCorp users.
 
 ## Ensuring accessible outcomes
 
-Because we are firm in our commitment to accessibility, we take practical steps to ensure that our outcomes are accessible. This means that we integrate accessibility at every step of our creation cycle, and include team education and training as part of that. In addition to formalizing our commitment to WCAG 2.1 AA conformance with an internal policy, here are the action steps we take:
+Because we’re firm in our commitment to accessibility, we take practical steps to ensure our outcomes are accessible. This means that we integrate accessibility at every step of our creation cycle. In addition to formalizing our commitment to WCAG 2.1 AA conformance with an internal policy, we take the following actionable steps:
 
 - Provide education and training for our team.
 - Design with accessibility as a core requirement.
 - Conduct design reviews specifically focused on accessibility.
 - Ensure our code renders to the browser in a conformant way.
-- Use the available accessibility automation (through [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint) and [ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing)).
+- Use available accessibility automation (through [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint) and [ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing)).
 - Manually test our code with assistive technologies.
 
 ## Compatibility expectations
 
 We intend to be compatible with the following assistive technologies:
 
 - Safari with VoiceOver on macOS
-- Chrome with Jaws on Windows
+- Chrome with JAWS on Windows
```

Comment:
Might be an opportunity to link out to these tools, but could also be a distraction

Reply:
I think it would be a distraction; additionally, the people who are most likely to read this statement will already know what it is. :)